README.md

```diff
@@ -14,215 +14,225 @@
   condition:
   - above: '0'
     condition: numeric_state
-    entity_id: sensor.smart_irrigation_daily_adjusted_run_time
+    entity_id: sensor.[zone_name]
   action:
-  - data: {}
+  - service: switch.turn_on
+    data: {}
     entity_id: switch.irrigation_tap1
-    service: switch.turn_on
   - delay:
-      seconds: '{{states("sensor.smart_irrigation_daily_adjusted_run_time")}}'
-  - data: {}
+      seconds: '{{states("sensor.[zone_name]")}}'
+  - service: switch.turn_off
+    data: {}
     entity_id: switch.irrigation_tap1
-    service: switch.turn_off
-  - data: {}
-    service: smart_irrigation.reset_bucket
+  - service: smart_irrigation.reset_bucket
+    data: {}
+    entity_id: sensor.[zone_name]
 ```
 
 [See more advanced examples in the Wiki](https://github.com/jeroenterheerdt/HAsmartirrigation/wiki/Automation-examples).
 
-### Step 4: configuring optional settings
-After setting up the component, you can use the options flow to configure the following:
-| Option | Description | Default |
-| --- | --- | --- |
-|Lead time|Time in seconds to add to any irrigation. Very useful if your system needs to handle another task first, such as building up pressure.| 0|
-|~~Change percentage~~|~~Percentage to change adjusted run time by. For example, you want to run 80% of the calculated adjusted run time, enter 80 here. Or, if you want to run 150% of the calculated adjusted run time, enter 150.~~|~~100~~|
-|Maximum duration| Maximum duration in seconds for any irrigation, including any `lead_time`. -1 means no maximum.|-1|
-|Show units|If enabled, attributes values will show units. By default units will be hidden for attribute values.|False|
-|Automatic refresh|By default, automatic refresh is enabled. Disabling it will require the user to call `smart_irrigation.calculate_daily_adjusted_run_time` manually.|True|
-|Automatic refresh time|Specifies when to do the automatic refresh if enabled.|23:00|
-|Initial update delay|Delay before first sensor update after reboot. This is useful if using sensors that do not have a status right after reboot.|30|
-|Coastal|If the location you are tracking is situated on or adjacent to coast of a large land mass or anywhere else where air masses are influenced by a nearby water body, enable this setting.|False|
-|Solar Radiation calculation|From v0.0.50 onwards, the component estimates solar radiation using temperature, which seems to be more accurate. If for whatever reason you wanted to revert back to the pre v0.0.50 behavior (which used a estimation of sun hours) disable this.|True|
-
-## Example behavior in a week
+## Example Behavior in a Week
 This [Wiki page](https://github.com/jeroenterheerdt/HAsmartirrigation/wiki/Example-behavior-in-a-week) provides insight into how this component should behave in certain weather conditions. With this you should be able to do a sanity check against your configuration and make sure everything is working correctly.
-## Available services
+
+## Available Services
```

Comment:
what is the difference between this and the other section about services?

Reply:
Removed